iNaturalist/src/main/java/org/inaturalist/android/ObservationListActivity.java

```diff
@@ -1050,7 +1054,6 @@ private void markAllObservationsAsRead() {
             // accidentally consider this an updated record)
             cv.put(Observation._SYNCED_AT, System.currentTimeMillis());
 
-            int count = getContentResolver().update(ContentUris.withAppendedId(Observation.CONTENT_URI, bc.getInt(Observation._ID)), cv, null, null);
```

Comment:
Why was this removed? I'm guessing because we didn't use the return value, but we still need to call the `update` method here.

Reply:
yup, my mistake